src/Abc.Zebus.MessageDsl/Generator/ProtoGenerator.cs

```diff
@@ -75,7 +75,7 @@ private void WriteEnum(EnumDefinition enumDef)
                 Writer.WriteLine("option allow_alias = true;");
 
             foreach (var member in enumDef.Members)
-                Writer.WriteLine("{0} = {1};", member.Name, member.InferredValueAsNumber ?? "TODO");
+                Writer.WriteLine("{0} = {1};",$"{enumDef.Name}_{member.Name}", member.InferredValueAsNumber ?? "TODO");
```

Comment:
Either use the old format syntax for everything (`{0} = {1}_{2};`) or the new one (`$"`), but please don't use both at the same time 😧 

Also, you deleted a space.